chapter-11/ticker/src/main/resources/application.properties

```diff
@@ -0,0 +1,4 @@
+mp.messaging.outgoing.ticks.connector=smallrye-kafka
+#mp.messaging.outgoing.ticks.connector=smallrye-amqp
```

Comment:
Remove it?

Reply:
This was done in all three services so that in the book I could tell them to switch dependencies and switch this config and then deploy the services for AMQP. Highlighting the fact the code is the same between Kafka and AMQP